components/gitpod-db/src/blocked-repository-db.spec.db.ts

```diff
@@ -35,26 +35,6 @@ class BlockedRepositoryDBSpec {
         expect(blockedRepository.blockUser).eq(true);
     }
 
-    @test(timeout(10000))
-    public async canDeleteABlockedRepository() {
-        const blockedRepository = await this.blockedRepositoryDb.createBlockedRepository("github.com/bob/*/", true);
-
-        const result = await this.blockedRepositoryDb.deleteBlockedRepository(blockedRepository.id);
-
-        expect(result).eq(true);
-        expect(await this.blockedRepositoryDb.findBlockedRepositoryByURL("github.com/bob/some-repo")).undefined;
-    }
-
-    @test(timeout(10000))
-    public async canNotDeleteABlockedRepositoryWithAnIdThatDoesNotExist() {
-        await this.blockedRepositoryDb.createBlockedRepository("github.com/bob/*/", true);
-
-        const result = await this.blockedRepositoryDb.deleteBlockedRepository(9999);
-
-        expect(result).eq(false);
-        expect(await this.blockedRepositoryDb.findBlockedRepositoryByURL("github.com/bob/some-repo")).not.undefined;
-    }
```

Comment:
Do you need to add these tests back in some form?

Reply:
I don't feel there's much value in it given our broken DB model. We still could do it, but I feel it's not worth the effort. :lotus_position: 
Once we have a working on, we definitely should definitely aim for testing this!